Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -172,6 +172,11 @@ class ClearPrivateDataTableViewController: UITableViewController {
              }
              }
              } else {*/
+            
+            // Reset Webkit configuration to remove data from memory
+            if !clear.filter({ $0 is CookiesClearable || $0 is CacheClearable}).isEmpty {
```

Comment:
Can be simplified to `!clear.contains(where: { $0 is CookiesClearable || $0 is CacheClearable })`

Reply:
Hehe missed my mind 👍

Reply:
Can even use `contains(where:)`

`if clear.contains(where: { $0 is CookiesClearable || $0 is CacheClearable}) {`

or if using actual values, this should even work:

`if clear.contains(where: [CookiesClearable, CacheClearable].contains) {`

Reply:
I think @iccub solution is cleanest/most obvious, but pointing out this would probably work too:
`two.contains(where: [CookiesClearable, CacheClearable].contains)`

Reply:
Also its a O(n^2) not that it matters in this case.